sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/tests/TimeSeriesInsightsInstancesTests.cs

```diff
@@ -125,26 +125,6 @@ await foreach (TimeSeriesInstance tsiInstance in getAllInstancesResponse)
                     tsiInstance.Should().NotBeNull();
                 }
                 numOfInstances.Should().BeGreaterOrEqualTo(numOfInstancesToSetup);
-
-                // Get search suggestions for the first instance
-                TimeSeriesId timeSeriesIdToSuggest = timeSeriesInstances.First().TimeSeriesId;
```

Comment:
removing this?

Reply:
Yes. I am removing the suggest API from the client library. Reason being is that it's too tied to the search API. And since we're not shipping the search API in preview, I am removing the suggest API